components/Header.js

```diff
@@ -1,20 +1,38 @@
 import styled from "styled-components";
+import Image from "next/image";
 
 const Headline = styled.header`
   position: fixed;
   top: 0;
-  left: 0;
   width: 100%;
-  margin: 0;
-  padding: 0.5em 1em;
-  text-align: center;
-  border: 3px solid black;
+  padding: 0.3em 0.8em;
   font-size: 2em;
   font-weight: bold;
-  background-color: #e3de4f;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #ded533;
```

Comment:
```suggestion
  background-color: var( --variable-name);
```
I guess it would be nice to use css variables for your colors, because you can make changes in one central place 
 when you like to change your app colors later.